src/models/environment.ts

```diff
@@ -31,6 +31,8 @@ const schema = (apiVersion: number): Joi.ObjectSchema => Joi.object().keys({
     .example(365)
     .min(1)
     .max(9999),
+  enforceTwoFactorAuthentication: Joi.boolean().allow(null).required().example(false)
```

Comment:
Third state (`null`) is not needed I think. Shall we therefore make this `boolean` only? Also for the TS interface on line 61, the supplier model and the routes.

Reply:
I have a migration for this and according to [requirements](https://github.com/withthegrid/platform#database-migrations), I need to have it null until I fill database with another migration